solrmonitor/solrmonitor.go

```diff
@@ -183,7 +191,14 @@ func (c *SolrMonitor) updateCollections(collections []string, isInit bool) {
 			wg.Add(1)
 			go func() {
 				defer wg.Done()
-				coll.start(isInit)
+				if err := coll.start(isInit); err != nil {
+					// Only possible to get an error here if isInit = true,
+					// and in this case we can propagate the error all the way back.
+					select {
+					case errChan <- err:
+					default:
```

Comment:
Only the first error will propagate an error back. Which is by design. But do we want to consider some sort of multi-error or maybe some sort of atomic counter to allow us to track how many collections failed to start?

Reply:
Sure! I'll at least keep a count and output num errors as part of the final log too.